### PR TITLE
🀄 Completed preliminary Chinese translation

### PR DESCRIPTION
**Pull request by BytinYarota**
(Except the salt name part actually

### CODE REVIEW

**Comment by BytinYarota** [comment]
The current work was mainly done by me, @YuriMontana and @yaodaosang collaboratively, with the help of other members of @CFPAOrg and some other helpful players.
Translating your mod was surely a challenge, with puns and technical terms tricky to deal with. I'm glad that we have came through all these, successfully completing the first stage of work.
The translation is still of poor quality, and we still have a long way to perfection. However, the current fruit definitely sets up a milestone, which we could be proud of.
Last but not least, sincere thanks to you for such an awesome mod, as well as immediate bug fixes, clear explanations and so on. Wish Destroy mod achieves its deserving success!🎉

File: src/main/resources/assets/destroy/lang/zh_cn.json [hunk 1]
```diff
@@ -156,269 +156,283 @@
...
-    "destroy.chemical.carbon_tetrachloride": "Carbon Tetrachloride",
-    "destroy.chemical.carbon_tetrachloride.iupac": "Tetrachloromethane",
-    "destroy.chemical.carboxylic_acid": "Carboxylic Acid",
-    "destroy.chemical.chloride": "Chloride",
-    "destroy.chemical.chlorine": "Chlorine",
-    "destroy.chemical.chloroaurate": "Chloroaurate",
-    "destroy.chemical.chloroaurate.iupac": "Tetrachloroaurate (III)",
-    "destroy.chemical.chloroauric_acid": "Chloroauric Acid",
-    "destroy.chemical.chloroauric_acid.iupac": "Tetrachloroauric (III) Acid",
+    "destroy.chemical.acetamide": "乙酰胺",
```
Comment (Catchmons) [inline src/main/resources/assets/destroy/lang/zh_cn.json]: 普通名字的话会不会用醋酸会好一点，系统命名法再用乙酸

Comment (BytinYarota) [reply src/main/resources/assets/destroy/lang/zh_cn.json]: 同意，另一处也是改的醋酸
以及把你add collaborators了

Comment (BytinYarota) [reply src/main/resources/assets/destroy/lang/zh_cn.json]: 已修改√

**Comment by petrolpark** [comment]
Splendid! Thank you so much for all of your help. If any of you who want the translator Badge don't have it yet, DM me on Discord at `petrolpark`. Once again, thank you!

**Pull request merged**